tests/TestCase/Console/ArgumentsTest.php

```diff
@@ -141,14 +141,14 @@ public function testGetOption(): void
         $options = [
             'verbose' => true,
             'off' => false,
-            'zero' => 0,
+            'zero' => '0',
```

Comment:
The `Arguments` class accepts an array of options, but these are expected to be parsed from command line so they must be string or bool at this point.